releasing-archway.md

```diff
@@ -0,0 +1,111 @@
+    emergency/critical fixes to the latest release
+
+### Major version timeline
+
+There is currently no mandated timeline for major versions beyond version 1.Y.Z, Y,Z >= 0. We haven't so far applied a rigorous interpretation of semantic
+versioning with respect to incompatible changes of any kind.
+
+TODO: Major versioning criteria need to be put up for discussion once Archway protocol reaches 1.Y.Z release.
+
+## Patch releases
+
+Patch releases are intended for critical bug fixes to the latest minor version,
+such as addressing security vulnerabilities, fixes to problems affecting a large
+number of users and severe problems with no workaround.
+
+They should not contain miscellaneous feature additions or improvements, and
+especially no incompatibilities should be introduced between patch versions of
+the same minor version (or even major version).
+
+Dependencies, such as cosmos-sdk or tendermint, should also not be changed unless
+absolutely necessary, and also just to fix critical bugs (so, at most patch
+version changes, not new major nor minor versions).
+
+## Network Upgrades
+
+- Upgrade documentation and a graceful migration path must be provided between all **consective** minor releases.
+  e.g. going from v1.3.Z, Z >= 0 to v1.4.Z, Z >= 0 must be accompanied by relevant documentation and automated migration path.
+- Network upgrade must also follow sequential minor releases. e.g. if a network is running v1.2.Z, Z >= 0, it must first
+  upgrade to v1.3.Z, Z >= 0 before upgrading to v1.4.Z, Z >= 0
+- Network upgrades may skip patch release e.g. a network running v1.2.1 may directly upgrade to v1.2.4
+- Network upgrades on archway-1 and constantine-1 must happen through software upgrade proposals.
+- Once a software upgrade proposal passes, all network operator are expected to upgrade to the exact
+  same archway protocol release within a reasonable timeframe.
+- Each new release must be adopted and tested on constantine-1 before being adopted on archway-1.
+- titus-1 is an unstable development network, with a single validator. Network upgrades on titus-1 are
+  expected to reset network state and start again from block 1 at any given time.
+
+### Open Questions in Network upgrades:
+
+- How to build artifacts for titus-1?
+  - Should there be a build on each new commit on main branch?
+  - use X.Y.0-{alpha,beta}.W, W > 0 (Branch: master)
+    Alpha and beta releases are created from tags on main branch branch and should be used to run titus-1
+
+### Important upgrade scenarios with examples
+
+The following scenarios have hypothetical versions and do not reflect versions of actual live networks
+
+#### constantine-1 needs a network upgrade that only applied to constantine-1
+
+Imagine we need to adjust 'const' (cost is the stake denom for constantine-1) token supply for constantine-1 and this requires an upgrade handler (TODO: Add link to upgrade handler description) specifically for constantine-1
```

Comment:
As mentioned in the above comment, for this situation, I think it would be best to export current `constantine-1` state and adjust the token supply and start `constantine-2` with the fixed state

Reply:
This network is being used by dapp developers as a canary network. Would doing a state export imply that dapp developers would need to deploy all the daps again?

Reply:
No idts. export and import also handles the contract binaries and its state as well

Reply:
Verifying the porposed workflow, will update the scenerio accordingly